TKeazirian.HTTPServer/Utils/FileUtility.cs

```diff
@@ -0,0 +1,23 @@
+namespace TKeazirian.HTTPServer.Utils;
+
+public static class FileUtility
+{
+    public static string GetPath()
+    {
+        string path = "";
+        var currentDirectory = Directory.GetCurrentDirectory();
+        if (currentDirectory == @"/Users/taylorkeazirian/Code/HTTPServer/TKeazirian.HTTPServer/bin/Debug/net6.0")
```

Comment:
this is still pretty smelly.  If I cloned your repo and ran tests, would they pass? or would I have to update this hardcoded path first?